lib/alephant/logger/cloudwatch.rb

```diff
@@ -31,9 +34,9 @@ def send_metric(name, value, unit, dimensions)
             :namespace   => namespace,
             :metric_data => [{
               :metric_name => name,
-              :value       => value,
-              :unit        => unit || "None",
-              :dimensions  => parse(dimensions || {})
+              :value       => value || @value,
+              :unit        => unit || @unit || "None",
+              :dimensions  => parse(dimensions || @dimensions || {})
```

Comment:
The third component of the `||`s won't be needed anymore as we now specify defaults